qiskit_aqua/multiclass/multiclass_quantumsvm/qkernel_svm_estimator.py

```diff
@@ -20,7 +20,7 @@
 from qiskit_aqua.svm.svm_qkernel import SVM_QKernel
 import numpy as np
 from qiskit_aqua.svm import (get_points_and_labels, optimize_SVM,
-                             kernel_join, entangler_map_creator)
+                              kernel_join, entangler_map_creator)
 
 class QKernalSVM_Estimator(Estimator, SVM_QKernel):
     def __init__(self, backend=None, shots=None):
```

Comment:
The backend and shots should be not set in the constructor. if your class is derived from QuantumAlgorithm, you should use `setup_quantum_backend` method to setup `backend` and `shots`.